forward_hand:
  ros__parameters:

    controller_name: "forward_hand_controller"
    wait_sec_between_publish: 3
    publish_topic: "/forward_hand_controller/commands"

    goal_names: ["pos1", "pos2", "pos3", "pos4"]
    pos1: [1.0, -0.01, 0.01]
    pos2: [0.0, 0.0, 0.0]
    pos3: [-1.57, -0.03, 0.03]
    pos4: [0.0, 0.0, 0.0]
